### DISCUSSION TITLE
en/chapter_hello_algo/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#995] [Announcements]
# en/chapter_hello_algo/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_hello_algo/

**Comment by gvkrwgfc** [comment]
The ordering of the code repository is too messy, and sometimes I can’t find the code for the chapter I’m looking for.

**Reply by krahets** [reply]
The name of the source file is marked above the code block, it is recommended to search for it. For example, in VS Code, if you want to find the `array.py` file, you can search:

<img width="364" alt="image" src="https://github.com/krahets/hello-algo/assets/26993056/7d4f56f6-de21-4362-b689-16202ed9078e">
